gui/src/installer/step/bitcoind.rs

```diff
@@ -200,7 +200,7 @@ impl InternalBitcoindConfig {
                 );
             } else if !prop.is_empty() {
                 return Err(InternalBitcoindConfigError::UnexpectedSection(
-                    "General section should be empty".to_string(),
+                    "bitcoin.conf general section should be empty".to_string(),
```

Comment:
It might be slightly better to mention the bitcoin.conf file in the `ReadingFile` error as returned by the `from_file` method below. You could also include the path to the file in this error variant's message. Then the "General section should be empty" message would be added at the end as additional info. For this, I think you'd need to wrap the error from `Self::from_ini(&conf_ini)` inside the `ReadingFile` error.

Reply:
Another option could be to mention the bitcoin.conf file in the `UnexpectedSection` error, which currently says "Unexpected section in file", and this message would also be shown for any other unexpected sections apart from the general section.